website/source/docs/job-specification/service.html.md

```diff
@@ -99,19 +99,22 @@ does not automatically enable service discovery.
 - `port` `(string: <required>)` - Specifies the label of the port on which this
   service is running. Note this is the _label_ of the port and not the port
   number. The port label must match one defined in the [`network`][network]
-  stanza.
+  stanza unless you're also using `address_mode="driver"`. Numeric ports may be
+  used when in driver addressing mode.
 
 - `tags` `(array<string>: [])` - Specifies the list of tags to associate with
   this service. If this is not supplied, no tags will be assigned to the service
   when it is registered.
 
 - `address_mode` `(string: "auto")` - Specifies what address (host or
   driver-specific) this service should advertise. `host` indicates the host IP
-  and port. `driver` advertises the IP used in the driver (e.g. Docker's internal
-  IP) and uses the ports specified in the port map. The default is `auto` which
-  behaves the same as `host` unless the driver determines its IP should be used.
-  This setting supported Docker since Nomad 0.6 and rkt since Nomad 0.7. It
-  will advertise the container IP if a network plugin is used (e.g. weave).
+  and port. `driver` advertises the IP used in the driver (e.g. Docker's
+  internal IP) and uses the container's port specified in the port map. The
+  default is `auto` which behaves the same as `host` unless the driver
+  determines its IP should be used.  This setting is supported in Docker since
+  Nomad 0.6 and rkt since Nomad 0.7. Nomad will advertise the container IP if a
+  network plugin is used (e.g. weave). See [below for
+  details.](#using-driver-address-mode)
```

Comment:
Did you see the other comment I had about making each of the modes sub list headers for clarity?